root/static/scripts/edit/MB/TextList.js

```diff
@@ -33,17 +33,21 @@ MB.Form.TextList = function (input) {
         return self;
     };
 
-    self.add = function (init_value) {
+    self.add = function (initValue) {
         $template.clone()
             .removeClass(template)
             .insertAfter($template
                 .parent()
                 .find('div.text-list-row')
                 .last())
             .show()
+<<<<<<< HEAD
+            .find('input.value').attr("name", input + '.' + counter).val(initValue)
+=======
             .find('input.value')
             .attr("name", input + '.' + counter)
             .val(init_value)
+>>>>>>> upstream/master
```

Comment:
Hi @ultra-programmer, I think you should resolve the conflict here first.

Reply:
Oops, well that's embarrassing... Sorry for the error. I will fix it right away!